### DISCUSSION TITLE
Timeline of the Civil War and Main Plot

### DISCUSSION THREAD

**Discussion by leklachu** [endless-sky/endless-sky#7140] [Lore]
I've been writing a campaign parallel to the main one, and using the events from main Free Worlds story to happen in the background. As I understand it, any official campaign will do something similar.

From https://github.com/endless-sky/endless-sky/wiki/StoryIdeas

>While the player's decisions are important enough to create large changes in the world, events such as the creation of the Oathkeepers, the attack on Poisonwood by the Alphas, the invasion of the Pug in order to stop the civil war, the deals that the Syndicate made with the Alphas, the Syndicate being the (true) instigators of the war, and many other things should remain constant among all the main plot campaigns. The circumstances of some of these events may change as a result of the player's actions, but they still occur. Just because the player joins the Syndicate (or some other faction) does not suddenly cause the universe to change such that the Syndicate weren't the ones who started the war.

It turns out the events in the gamefiles are... kind of logical. In the hopes it's helpful to anyone else making campaigns (e.g. the official Navy/Syndicate campaigns), here are the files I've made to help me navigate this. They're only for the Reconciliation branch so far, because I'm not expecting to deal with the Checkmate branch for a while! But if it's useful to people, I'm sure I can go through and finish that.

Also: the Free Worlds campaign is being reworked: https://github.com/endless-sky/endless-sky/pull/7000 - which gives a chance to:

- fix general mission events so they don't trigger Free-Worlds-specific things
- change all the event names and muck up my campaign!

Obviously the first I'd like, and the second, if it happens, I'd like to know the details so I can fix my campaign. Maybe this discussion can serve as a place to record those and any future changes.

**Comment by leklachu** [comment]
Flowchart of the FW campaign, following Reconciliation branch.

The SVG can be downloaded then opened. (Clicking it directly seems to display a mess.) PNG included for convenience

![endless-sky_story-flow](https://user-images.githubusercontent.com/22275154/185368107-05ab7064-3ca8-4906-a2a1-59c79c32aa98.svg)

![endless-sky_story-flow](https://user-images.githubusercontent.com/22275154/185368121-d388eb40-1f2f-4c5e-9f3e-e76c7a7addc8.png)

**Comment by leklachu** [comment]
A list of the events mostly from the data/events.txt file, arranged to show the plot as it develops. Following Reconciliation branch, but the events for Checkmate are in there too. I haven't yet checked the Reconciliation events that they don't do any FW-specific stuff.

[events.md](https://github.com/endless-sky/endless-sky/files/9372467/events.md)

---

Notes: These are the events from data/human/events.txt, laid out to show how the events of the war unfold, which might be the same from any parallel campaign.

- Free Worlds-specific events are included for reference, but in brackets, like (joined the free worlds)
- Events within the Free Worlds mission files should be included if relevant to other campaigns. Marked like [free worlds start]. They're mostly setting up temporary fleets for while a mission is in progress.
- Some events have mixed general with Free Worlds-specific content, or have a Free Worlds-specific mission that triggers automatically from the event. These are marked with !!FW-specific content!!
- Where events are triggered with a delay in the Free Worlds storyline, this is marked.

---

# The Beginning

remembrance day, 3014-02-19

- mission "remembrance day"

Independence:

- war begins, 3014-07-04
	+ At this point, Navy and FW still don't fight.
	+ mission "event: war begins"
- initial deployment 1, 3014-07-24
- initial deployment 2, 3014-08-14
- initial deployment 3, 3014-08-29
	+ mission "Southern Carriers Trigger"
	+  90d: southern carriers 1
	+ 180d: southern carriers 2
	+ 270d: southern carriers 3
	+ 360d: southern carriers 4
- initial deployment 4, 3014-09-17

# War starts in earnest... if you join

Navy takes over Kornephoros, and fighting starts. Free Worlds take it back.

- capture of Kornephoros
- start of hostilities
	+ Republic/FW become hostile
	+ Player/Republic become hostile
		* !!FW-specific content!!
- (joined the free worlds)
- recapture of Kornephoros
- (temporary ceasefire) (start of hostilities)
- oathkeepers founded ~ after 50 days
	+ [free worlds start] mission: "oathkeepers founded"
		* !!FW-specific content!!

Free Worlds negotiates with Syndicate and Tarazed, and prepares to defend itself

- Tarazed neutrality
- plasma turret available ~ 50 days after testing
	+ [free worlds start] mission: "plasma turret available"
		* !!FW-specific content!!
	+ [free worlds side plots] mission: "FW Flamethrower 1"
		* !!FW-specific content!!
		* flamethrower available ~ 30 days
		* OR at latest shortly after southern worlds join the Free Worlds

Free Worlds considers about pirates.

- fw conservatory founded ~ 30 days if did side-plots missions
	+ at the latest this is founded around the time Albatross joins Free Worlds

A good, democratic Captain will help Thule

- Thule becomes independent

OR they'll attck Bloodsea or Greenrock

- fw supressed Bloodsea
- fw abandoned Bloodsea ~ 70 days
- OR: fw suppressed Greenrock
- fw abandoned Greenrock ~ 40 days

50 days after the pirate stuff, some southern worlds join the Free Worlds

- navy occupying the south
	+ [free worlds middle] mission: "FW Diplomacy 1"
		* !!FW-specific content!!
- catalytic ramscoop available
	+ [free worlds side plots] mission: "FW Wolf Pack 1"
		* !!FW-specific content!!
		* fw gets catalytic ramscoop ~ 30 days
- (fw safe passage starts) ~ 24 days (fw safe passage ends)
- navy using mark ii ships
- (gunboats in Alnasl) (fw safe passage ends) (normal in Alnasl)
- fw southern expansion
- OPTIONAL fw prison on Clink
	+ mission: "FW Clink Prison Closes"
	+ fw prison on Clink closes

After another 50 days, northern worlds want to join Free Worlds

- fw occupying the north
	+ [free worlds middle] mission: "FW Southern Break Ends"
	+ [free worlds middle] mission: "FW Northern 1"
		+ !!FW-specific content!!
- fw northern expansion

The Alphas capture Poisonwood, and the Free Worlds fight back, with or without Danforth's help

- alphas catpure Poisonwood
- liberation of Poisonwood
	+ (death of nguyen ~ 35 days)
		* In [wanderers start] this is referenced as whether you know Nguyen before he died; but it's assumed he died anyway
		* [free worlds middle] mission: "FW death of Nguyen"
			- !!FW-specific content!!
	+ !!FW-specific content!! in Poisonwood spaceport description
- OR Poisonwood reverts to Republic
	+ !!FW-specific content!! in Poisonwood spaceport description

The Free Worlds goes to negotiate with Blootsea

- bloodsea joins free worlds
- bloodsea spaceport completed ~ 100 days


OR batter them into submission

- battle for bloodsea
- bloodsea independent

By which time dreadnoughts are ready to sell

- dreadnoughts for sale
- dreadnought deployment

Lastly, Albatross is approached by the Free Worlds

- albatross joins free worlds

Rand and neighbours get excited and want to join too! But the Navy cut them off

- fw expanded and cut

But... some dude from the Syndicate comes with confession

# It's all their fault!

Free Worlds rescues Sawyer, the Syndicate defector, and turns on the Syndicate.

- fw at war with Syndicate
- fw armstice
- fw tarazed republic

Starts working things through with Navy Intelligence

- deep sky tech available
- if karma > 0: fw tarazed joins
- if karma > 0: navy out of rastaban ~ 5 days
	+ otherwise at the end

Sets up an embassy between the Republic and Free Worlds, and takes a break to prepare for the arrest of Alexander Soylent

- stack core for sale ~ if you do the quest here; otherwise at the end

The Syndicate panics. And the Pug... are smug

- syndicate occupies sol
- pug invasion
- pug invasion 2 ~ 6 days
- pug invasion 3 ~ 10 days
- pug invasion 4 ~ 14 days

When the Free Worlds captain goes to meet the Syndicate, they don't want to fight any more... and the extremists have set up on Algenib

- fw syndicate welcoming

The Free Worlds goes around trying to make sense of everything, and eventually everyone agrees to kill the Pug.

- at war with the pug
- battle for delta capricorni
- reconnected delta capricorni
- battle for altair
- liberation of altair
- reconnected altair

At last the final showdown... with empty space. And the galaxy can 'come together' again. There's also a, um, huh. Where does that go??

- pug flee
- pug territory liberated
- pug link restoration 1 ~ 10 days
- pug link restoration 2 ~ 25 days
- pug link restoration 3 ~ 45 days
- pug link restoration 4 ~ 60 days

Now to take out our angst on the real culprits: the extremists and alphas.

- battle of algenib
- navy occupies algenib
- navy done with algenib ~ 50 days

When it's all over, the Navy start rebuilding Geminus

- mission "Geminus Rebuilt"
- geminus rebuilt ~ 90 days

---

# OR Free Worlds can keep fighting the Navy. Serves the Republic right for being so pig-headed.

- fwc southern battle
- fwc southern liberation
- fwc attack kaus boralis
- fwc capture kaus borealis
- fwc attack cebalrai
- fwc capture cebalrai
- fwc solace has nukes
- fwc defend cebalrai
- fwc defended cebalrai
- fwc attack menkent
- fwc capture menkent
- fwc attack vega
- fwc pug invasion
- fwc pug peaceful
- fwc navy retakes cebalrai
- fwc peace with the navy
- fwc defend oblivion
- fwc battle for rasalhague
- fwc liberation of rasalhague
- fwc reconnect ascella
- fwc reconnect zeta aquilae
- fwc liberation of vega
- fwc reconnect vega
- fwc pug defeated
- fwc link restoration 1
- fwc link restoration 2
- fwc link restoration 3
- syndicate tech available

**Comment by IsocratesGrey** [comment]
Thank you so much for doing that @leklachu. I took what you created and turned it into a main plot spreadsheet to help people figure out how to plot an alternative questline that still follows the same structure. The table is below, but the uploaded file also has a quest line structure for all of the game except for the Remnant (I hope to add them later). 

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/narney/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/narney/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



Core   Event | Free Worlds Chosen | FW Quest Line | Republic Chosen | Syndicate Chosen | Story Beats | Plotline Trigger Current
-- | -- | -- | -- | -- | -- | --
Remeberance Day |   | Intro |   |   | Introduce Alphas | Date
War Begins |   | Free Worlds Prologue |   |   | War starts | Date
Initial Deployment 1 |   | Free Worlds Prologue |   |   | Prologue | Date
Initial Deployment 2 |   | Free Worlds Prologue |   |   | Prologue | Date
Initial Deployment 3 |   | Free Worlds Prologue |   |   | Prologue | Date
Initial Deployment 4 |   | Free Worlds Prologue |   |   | Prologue | Date
capture of Kornephoros |   | Free Worlds Prologue |   |   | Inciting Incident | Chosen Sides
start of hostilities |   | Free Worlds Prologue |   |   | Inciting Incident | Chosen Sides
  | recapture of Kornephoros | Defend the Free Worlds |   |   | Inciting Incident | Plot Chapter 1
  | oathkeepers founded | Defend the Free Worlds |   |   | Inciting Incident | Plot Chapter 1
  | Tarazed neutrality | Defend the Free Worlds |   |   | Player meeting major players in campaign | Plot Chapter 1
  | fw conservatory founded | Defend the Free Worlds |   |   | Player impacts galaxy | Plot Chapter 1
  | Thule, Bloodsea, Greenrock event | Defend the Free Worlds |   |   | Player impacts galaxy | Plot Chapter 1
navy occupying the south |   | Defend the Free Worlds |   |   | Close of chapter 1 | Plot Chapter 1
navy using mark ii ships |   | Freeworlds Expansion, Alphas |   |   | Open of chapter 2 | Plot Chapter 2
  | fw southern expansion | Freeworlds Expansion, Alphas |   |   | Player climbs the ranks | Plot Chapter 2
  | fw prison on Clink | Freeworlds Expansion, Alphas |   |   | Player climbs the ranks | Plot Chapter 2
  | fw occupying the north | Freeworlds Expansion, Alphas |   |   | Player climbs the ranks | Plot Chapter 2
  | fw northern expansion | Freeworlds Expansion, Alphas |   |   | Player climbs the ranks | Plot Chapter 2
alphas catpure Poisonwood |   | Freeworlds Expansion, Alphas |   |   | Return of the Alphas | Plot Chapter 2
liberation/reversion of Poisonwood |   | Freeworlds Expansion, Alphas |   |   | Return of the Alphas | Plot Chapter 2
  | (death of nguyen ~ 35 days) | Freeworlds Expansion, Alphas |   |   | Making the consequences real | Plot Chapter 2
  | Bloodsea joins/taken by FW | Freeworlds Expansion, Alphas |   |   | Player impacts galaxy | Plot Chapter 2
dreadnought deployment |   | Freeworlds Expansion, Alphas |   |   | Faction gaining strength | Plot Chapter 2
  | albatross joins free worlds | Freeworlds Expansion, Alphas |   |   | Player climbs the ranks | Plot Chapter 2
  | fw expanded and cut | Freeworlds Expansion, Alphas |   |   | Player climbs the ranks | Plot Chapter 2
  | FW Defend New Tibet | Freeworlds Expansion, Alphas |   |   | Climax of chapter | Plot Chapter 2
  | fw at war with Syndicate or republic | Freeworlds Expansion, Alphas |   |   | Close of chapter 2: a new war | Plot Chapter 2
  | fw Syndicate/Republic war | Fight the Republic/Syndicate |   |   | Chapter 3A: Player fills major role in the war | Plot Chapter 3
  | syndicate occupies/FW attacks sol | Fight the Republic/Syndicate |   |   | Someone attacks Sol (end ch 3A) | Plot Chapter 3
pug invasion |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
pug invasion 2 ~ 6 days |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
pug invasion 3 ~ 10 days |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
pug invasion 4 ~ 14 days |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
  | fw syndicate welcoming | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
at war with the pug |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
battle for delta capricorni |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
reconnected delta capricorni |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
battle for altair |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
liberation of altair |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
reconnected altair |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
pug flee |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
pug territory liberated |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
pug link restoration 1 ~ 10 days |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
pug link restoration 2 ~ 25 days |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
pug link restoration 3 ~ 45 days |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
pug link restoration 4 ~ 60 days |   | Repel Pug Invasion |   |   | War with the Pug (unified across the factions?) | Plot Chapter 3
battle of algenib |   | Repel Pug Invasion |   |   | Ch 3 Climax: Defeat the Alpha Threat | Plot Chapter 3
navy occupies algenib |   | Repel Pug Invasion |   |   | Ch 3 denouement | Plot Chapter 3
navy done with algenib ~ 50 days |   | Repel Pug Invasion |   |   | Ch 3 denouement | Plot Chapter 3
mission "Geminus Rebuilt" |   | Repel Pug Invasion |   |   | Ch 3 denouement | Plot Chapter 3
geminus rebuilt ~ 90 days |   | Repel Pug Invasion |   |   | Ch 3 denouement | Plot Chapter 3
  |   | Free Worlds Epilogue |   |   | Main Plot Epilogue | Epilogue



</body>

</html>

[Endless Sky Plot Book.xlsx](https://github.com/endless-sky/endless-sky/files/10118157/Endless.Sky.Plot.Book.xlsx)
